modules/virt-deprecated-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-deprecated-tasks_{context}"]
+= Removing deprecated or unused resources
+
+* The Tekton-tasks-operator (TTO) has been deprecated and removed in {VirtProductName} 4.17. You can clean up any remaining resources associated with the operator by running the following command.
+
+.Procedure
+
+. The following command removes any remaining tekton-related resources from the cluster:
```

Comment:
```suggestion
. To remove any remaining Tekton-related resources from the cluster, run the following command:
```

Reply:
See this style resource regarding commands: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command